docs/pages/reference/access-controls/access-monitoring-rules.mdx

```diff
@@ -0,0 +1,271 @@
+| Slack               | Email: `user@example.com`<br/>Channel: `example-channel`                              |
+
+### Example notification routing rule
+
+If the following rule is configured, then anytime an Access Request for the
+`access` role is created, a notification will be sent to the `#teleport-access`
+Slack channel.
+
+```yaml
+kind: access_monitoring_rule
+version: v1
+metadata:
+  name: slack-notifications
+spec:
+  subjects:
+  - access_request
+  condition: contains_any(access_request.spec.roles, set("access"))
+  notification:
+    name: slack-default
+    recipients:
+    - teleport-access
+```
+
+## Automatic review rules
+
+Administrators can configure Access Monitoring Rules to automatically review an Access
+Request when conditions are met.
+
+To enable automatic reviews, set `spec.desired_state` to `reviewed` and define
+`automatic_review`.
+
+### Review decision
+
+The `automatic_review.decision` option can be either `APPROVED` or `DENIED`.
+
+Notes:
+- Automatic reviews behave like regular reviews. They do not directly change the
+  state of the request and respect the required review thresholds.
+- If multiple automatic review rules match an Access Request, `DENIED` rules
+  take precedence.
+
+### Example automatic review rule
+
+If the following rule is configured, then anytime an Access Request for the
+`access` role is created by a user with the `team: sre` trait, it will be
+automatically approved.
+
+```yaml
+kind: access_monitoring_rule
+version: v1
+metadata:
+  name: sre-automatic-approval
+spec:
+  subjects:
+  - access_request
+  condition: |-
+    contains_all(set("access"), access_request.spec.roles) &&
+    contains_any(user.traits["team"], set("sre"))
+  desired_state: reviewed
+  automatic_review:
+    integration: builtin
+    decision: APPROVED
+```
+
+## Condition
+
+The `condition` field is a predicate expression that evaluates to a boolean value
+and determines which Access Requests the rule applies to.
+
+Accepted fields within the condition predicate expression:
+| Field                                   | Description                                    |
+| --------------------------------------- | ---------------------------------------------- |
+| access_request.spec.roles               | The set of roles requested.                    |
+| access_request.spec.suggested_reviewers | The set of reviewers specified in the request. |
+| access_request.spec.system_annotations  | A map of system annotations on the request.    |
+| access_request.spec.user                | The requesting user.                           |
+| access_request.spec.request_reason      | The request reason.                            |
+| access_request.spec.creation_time       | The creation time of the request.              |
+| access_request.spec.expiry              | The expiry time of the request.                |
+| user.traits                             | A map of traits of the requesting user.        |
+
+Examples:
```

Comment:
This looks a little cluttered to me: 
<img width="704" alt="image" src="https://github.com/user-attachments/assets/fb4c98a9-10d6-49b2-a955-b00c4e78fc23" />

I would use code fences for the example snippets make the text to the right of each dash either a comment above each snippet or regular text preceding each snippet.